lambdas/indexer/.chalice/config.json.template.py

```diff
@@ -11,7 +10,8 @@
     "api_gateway_stage": config.deployment_stage,
     "manage_iam_role": False,
     "iam_role_arn": "${var.role_arn}",
-    "environment_variables": aws.lambda_env,
+    "environment_variables": config.lambda_env(("${var.azul_es_domain.endpoint}", 443),
```

Comment:
This would be more readable if the args were passed as keywords with hanging indent.